tests/swift-sdk-swift-tests/InAppPresenterTests.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Created by Jay Kim on 5/5/20.
+//  Copyright © 2020 Iterable. All rights reserved.
+//
+
+import UIKit
+import WebKit
+import XCTest
+
+@testable import IterableSDK
+
+class InAppPresenterTests: XCTestCase {
+    static let delayTimerInterval = 0.75
```

Comment:
We should make this parameterized. So that we can test with smaller/larger values.